beetsplug/bpd/gstplayer.py

```diff
@@ -21,16 +21,21 @@
 
 import sys
 import time
-import gobject
 import thread
 import os
 import copy
 import urllib
 
-import pygst
-pygst.require('0.10')
-import gst  # noqa
+import gi
+from gi.repository import GObject, Gst
 
+gi.require_version('Gst', '1.0')
+
+GObject.threads_init()
```

Comment:
Apparently, `threads_init` is no longer necessary and emits a warning now in modern GObject versions. This Pull Request shows us resolving this in the `audioread` package: https://github.com/beetbox/audioread/pull/30/files